src/formatDate.js

```diff
@@ -50,7 +50,42 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDate = date.split(fromFormat[3]);
```

Comment:
good, but are you sure that separator always be on this index?

Reply:
agree, thanks)